.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - '*.*.*'
+
+jobs:
+  test:
+      runs-on: macos-latest
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Make
+        run: make
+
+      - name: Run tests
+        run: make unit-tests
+
+  release:
+    needs: ["test"]
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Get tag version
+      id: version
+      run: echo ::set-output name=TAG_VERSION::${GITHUB_REF#refs/*/}
+
+    - name: Release
+      uses: softprops/action-gh-release@v0.1.7
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        name: Version ${{ steps.version.outputs.TAG_VERSION }}
```

Comment:
The automatic release is great!
I preferred the full name "Snowplow C++ Tracker v{{version}}" but I see that there isn't a common approach among the tracker repos.
I can agree with this format "Version {{version}}" but we need to enforce it in the other remaining trackers (mobile ones for example) as I'd like to have a coherent approach for all of them rather than many different ways.

For a future improvement...
it would be nice to add the release notes. In the tracker contribution guide we suggest to write them in the Release PR. It would be nice to copy that one here or make use of [this](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes).